classes/class15/README.md

```diff
@@ -1 +1,72 @@
+They list these four piorities as being central to the agile methodology: 
+
+- Individuals and interactions over Processes and tools
+- Working software over Comprehensive documentation
+- Customer collaboration over Contract negotiation
+- Responding to change over Following a plan
+
+More specifically, there are 12 principles that also help define agile: 
+
+- Customer satisfaction by rapid delivery of useful software
+- Welcome changing requirements, even late in development
+- Working software is delivered frequently (weeks rather than months)
+- Close, daily cooperation between business people and developers
+- Projects are built around motivated individuals, who should be trusted
+- Face-to-face conversation is the best form of communication (co-location)
+- Working software is the principal measure of progress
+- Sustainable development, able to maintain a constant pace
+- Continuous attention to technical excellence and good design
+- Simplicity—the art of maximizing the amount of work not done—is essential
+- Self-organizing teams
+- Regular adaptation to changing circumstance
+
+##Scrum
+As mentioned ealier, agile is more of a mindset and methodology than an actual set of instructions on exactly what to do. A very popular addition to agile that brings in more specific instructions is the scrum framework. In fact, scrum is used so often with agile that components of one are often mistaken for components of the other. What srum adds is a set of roles and events to your development process that are meant to help maximize your efficiency. It is worth noting that while there are "official" scrum methods, the actual use and implementation varies a good amount from company to company and even team to team. 
+
+https://www.youtube.com/watch?v=oyVksFviJVE
+
+###Stories
+A big part of scrum is breaking down the tasks of a project into what are called stories. 
+These stories could be features, bugs, learning about something, and so on. 
+Each story is given a point value which is meant to be an indication of how much time it will take.
+The meaning of points varies from team to team, but as long as everyone on the team has a good idea, it doesn't matter too much. 
+Some places say one point is 1 day or define 3 points to be some basic task such as adding a button to a page.
+There is a little ceremony that happens where all the software developers on a team get together and estimate stories in the backlog.
+A story is chosen and the developers talk about it very briefly. 
+They then each pick an estiamte for the story and if everyone agrees that becomes the estimation. 
+If not, they discuss why and quickly settle on an estimate.
+The offical scrum teachings state that a story can take only valeus of 0, 1, 2, 3, 5, 13, 20, 40, 100. 
+This is meant to keep you from nitpicking over point estimations. 
+Stories should be ordered in the backlog in order of importance to the project. 
```

Comment:
I will add that in, but at my previous work places we estimated first so that the PM could help prioritize the stories based on the customer benefit to work amount ratio.